src/components/circle.vue

```diff
@@ -1,24 +1,51 @@
 /* vim: set softtabstop=2 shiftwidth=2 expandtab : */
 
+<template>
+</template>
+
 <script>
 
 import _ from 'lodash';
-
 import eventBinder from '../utils/eventsBinder.js'
 import propsBinder from '../utils/propsBinder.js'
-import MapElementMixin from './mapElementMixin';
+import MapComponent from './mapComponent';
 import getPropsValuesMixin from '../utils/getPropsValuesMixin.js'
 
+const circleProps = {
```

Comment:
all this ObjProps are created to be compatible and reuse propsBinder.
to know what can be two way bind and not.